charts/monochart/templates/deployment.yaml

```diff
@@ -182,29 +179,29 @@ spec:
       {{- if .Values.deployment.volumes }}
         {{- toYaml .Values.deployment.volumes | nindent 6 }}
       {{- end }}
-      {{- if .Values.deployment.ephemeral.enabled }}
-      - name: scratch
+      {{- if .Values.persistence }}
+      {{- if .Values.persistence.enabled }}
+      - name: {{ .Values.persistence.storageName }}
+      {{- if and (hasKey .Values.persistence "ephemeral") .Values.persistence.ephemeral }}
```

Comment:
In general I'm good with the changes, just the question: why does it have to be for example **.Values.persistence.enabled** instead of **.Values.deployment.persistence.enabled** ?